### PR TITLE
daemon: check plugins status before response

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

**Comment by yorkie** [comment]
@FeelyChau May I ask you to add a test for this patch?

**Comment by FeelyChau** [comment]
> @FeelyChau May I ask you to add a test for this patch?

Test cases added.

File: packages/daemon/test/controller/job.test.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    });
+    app.mockClassFunction('pipelineService', 'fetchPlugins', async (pipeline: any): Promise<any[]> => {
+      assert.deepEqual(pipeline, mockPipeline);
+      throw createHttpError(HttpStatus.NOT_FOUND, 'mock error');
+    });
+    app.mockClassFunction('pipelineService', 'createJob', async (pipelineId: string): Promise<any> => {
+      assert.equal(pipelineId, 'id');
+      return mockJob;
+    });
+    app.mockClassFunction('pipelineService', 'runJob', async (job: any, pipeline: any, plugins: any[], log: any): Promise<any> => {
```
Comment (yorkie) [inline packages/daemon/test/controller/job.test.ts:105]: What's the strategy of restoring the app?

Comment (FeelyChau) [reply packages/daemon/test/controller/job.test.ts:105]: I can't find the document for this, but as I know, the services and controllers in mock cases are the same as IOC lifecycle.

Comment (FeelyChau) [reply packages/daemon/test/controller/job.test.ts:105]: We should be careful when mock a singleton object.

**Review by yorkie — APPROVED** [review]

**Pull request merged**